apps/docs/content/components/SectionIntro.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: SectionIntro
+status: Experimental
+source: https://github.com/primer/brand/tree/main/src/SectionIntro/SectionIntro.tsx
+storybook: '/brand/storybook/?path=/story/components-SectionIntro--playground'
+description: Use SectionIntro to provide a title and optional description paragraph and action to a new section in the page.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {SectionIntro} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<SectionIntro>
+  <SectionIntro.Heading>Focus on solving bigger problems</SectionIntro.Heading>
+  <SectionIntro.Description>
+    Spend less time creating boilerplate and repetitive code patterns, and more
+    time on what matters: building great software. Write a comment describing
+    the logic you want and GitHub Copilot will immediately suggest code to
+    implement the solution.
+  </SectionIntro.Description>
+  <SectionIntro.Link href="#">Explore docs</SectionIntro.Link>
+</SectionIntro>
+```
+
+### Alignment
+
+The content alignment can be changed using the `align` prop on the root `SectionIntro`. This can be configured to either `start` or `center`.
+
+```jsx live
+<SectionIntro align="center">
+  <SectionIntro.Heading>Focus on solving bigger problems</SectionIntro.Heading>
+  <SectionIntro.Description>
+    Spend less time creating boilerplate and repetitive code patterns, and more
+    time on what matters: building great software. Write a comment describing
+    the logic you want and GitHub Copilot will immediately suggest code to
+    implement the solution.
+  </SectionIntro.Description>
+  <SectionIntro.Link href="#">Explore docs</SectionIntro.Link>
+</SectionIntro>
+```
+
+## Component props
+
+### SectionIntro <Label>Required</Label>
+
+| name        | type                       | default     | required | description                                                   |
+| ----------- | -------------------------- | ----------- | -------- | ------------------------------------------------------------- |
+| `children`  | `ReactNode`, `ReactNode[]` | `undefined` | `true`   | Content to include within the section intro component         |
+| `align`     | `start`, `center`          | `start`     | `true`   | The layout applied to the content wihin the section component |
+| `className` | `string`                   |             | `false`  | SectionIntro custom class                                     |
+
+### SectionIntro.Heading <Label>Required</Label>
+
+Required node that is used to provide a heading for the section.
```

Comment:
Shall we mention here that we forward all props from `Heading`, like `weight` too? Maybe cross-link for convenience too.